README.md

```diff
@@ -30,8 +28,9 @@ This is an example of the declarative way of instantiating an error message that
 <div class="o-message o-message--alert o-message--error" data-o-component="o-message">
 	<div class="o-message__container">
 		<div class="o-message__content">
-			<p class="o-message__highlight">Something went wrong!
-				<span class="o-message__detail">The quick brown fox did not jump over the lazy dogs.</span>
+			<p class="o-message__content--main">
```

Comment:
If we're making a breaking to tidy up the html interface is it possible to cut out a wrapper? So the content isn't nested as much.

The new class names are BEM modifiers of `o-message__content` when they should be elements.